CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 3.1.1 (12.08.2020)
```

Comment:
Nah, let's not bump the version. The PR changes README only, there're no bugfixes.

Reply:
Done.

Reply:
🚀